packages/csv-stringify/lib/api/normalize_options.js

```diff
@@ -109,6 +109,28 @@ const normalize_options = function(opts) {
   }else{
     // todo
   }
+  // Normalize option `headers_as_comment`
```

Comment:
Could you normalize both options, `comment` and `header_as_comment` separately where `comment` default to `#`.